### DISCUSSION TITLE
Is it possible to collimate a radionuclide source?

### DISCUSSION THREAD

**Discussion by ProfLeao** [nrc-cnrc/EGSnrc#738] [Questions] [answered]
Hello @ftessier. How do you assess the complexity of implementing the support of sources derived from egs_collimated_source to also be able to use radionuclide spectra? I'm modelling a radiator for which the computational efficiency of the simulation has been very low due to the small solid angle of the aperture cone and the isotropic emissivity profile of the source. A collimated font would be great in this case. If it's not too complex, I could try to complete the task.

**Comment by ftessier** [comment]
@rtownson, do we have a radionuclide collimated source?

**Reply by rtownson** [reply]
@ProfLeao Yes, we do already have a radionuclide collimated source! It is identical in function to `egs_collimated_source`. See here:

https://nrc-cnrc.github.io/EGSnrc/doc/pirs898/classEGS__RadionuclideSource.html

If you are using `egs_radionuclide_spectrum`, please keep in mind that it must be used with `egs_radionuclide_source`. This source has both isotropic and collimated options. Be aware that the collimated source has a different normalization, depending on the solid angle. There is a bit of explanation of this in the getting-started manual as well as in the documentation for `egs_collimated_source`.

**Reply by ProfLeao** [reply]
I'm studying a Cesium 137 irradiator that uses a very narrow cone for exposure. I will reproduce some exposures in it, however, due to geometric conditions the statistic of particles that reach the sensitive volume is quite small. A collimated source of Cs-137 would help a lot.
![unnamed](https://user-images.githubusercontent.com/46849605/122583755-f8bb5900-d02f-11eb-8297-9bc1c9e6d3ae.png)

**Reply by ftessier** [reply]
@rtownson I try to mark this as *answered,* but then it says that it was answered by me!! Oh well 😄 . So I understand now that we ought to provide answers as separate comments, rather than in a reply to a comment. Do you mind duplicating your reply as a new comment, so that I can attribute it correctly?

**Reply by rtownson** [reply]
@ProfLeao this seems like a reasonable situation for the collimated source, as long as you are aware of the approximation.

**Reply by ProfLeao** [reply]
Yes, I'm aware of the approximation.
I'm trying now.

**Reply by ProfLeao** [reply]
Sorry for the delay. Worked perfectly.
![cesio](https://user-images.githubusercontent.com/46849605/123440341-0d0bd280-d5a9-11eb-910e-0654b76f7a18.jpeg)

**Comment by rtownson** [comment, marked as answer]
@ProfLeao Yes, we do already have a radionuclide collimated source! It is identical in function to egs_collimated_source. See here:

https://nrc-cnrc.github.io/EGSnrc/doc/pirs898/classEGS__RadionuclideSource.html

If you are using egs_radionuclide_spectrum, please keep in mind that it must be used with egs_radionuclide_source. This source has both isotropic and collimated options. Be aware that the collimated source has a different normalization, depending on the solid angle. There is a bit of explanation of this in the getting-started manual as well as in the documentation for egs_collimated_source.
